### ISSUE TITLE
Method param with class constant as default value

### ISSUE THREAD

**Issue by Bob4ever** [goaop/parser-reflection#43]
I'm actually trying to use the parser-reflection for analyzing our code but I may found a bug:

```
public function addStamp($positionOrConfig = self::POSITION_LEFT_TOP)
```

When I try to analyze this code with ReflectionParameter::getDefaultValueConstantName I would expect "self::POSITION_LEFT_TOP" but instead I get "NULL". When I use the php-ReflectionParameter everything works like expected.

**Comment by aik099** [comment]
Nice find. Would you please send a PR with a failing test case and a fix?

**Comment by lisachenko** [comment]
Hi, it's definitely a bug. https://3v4l.org/BHbtR. Need to add support for this into the `NodeExpressionResolver` class.

**Comment by Bob4ever** [comment]
I created a pull request for this one #44

**Comment by lisachenko** [comment]
Resolved in #44. Thanks!
